data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/ConditionalExpressionEvaluator.java

```diff
@@ -0,0 +1,33 @@
+package org.opensearch.dataprepper.expression;
+
+import com.amazon.dataprepper.model.event.Event;
+import org.antlr.v4.runtime.tree.ParseTree;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+/**
+ * Public class that {@link com.amazon.dataprepper.model.processor.Processor},
+ * {@link com.amazon.dataprepper.model.sink.Sink} and data-prepper-core objects can use to evaluate statements.
+ */
+@Named
+public class ConditionalExpressionEvaluator implements ExpressionEvaluator<Boolean> {
```

Comment:
Does this class have to be public? I'm guessing that clients will get by Dependency Injection of an `ExpressionEvaluator`.